solutions/telemetry/console/consumer/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are we using Maven for a plain Java app? Perhaps to me is a little bit of overhead. Can we stick to a plain Java file that we can compile with `javac`?

Another option is perhaps to use submodules, one for the consumer, and another for the producer, and avoid adding twice `mvnw` and related files. (I think that we can make it work)

Reply:
mmm, I used maven to handle the kafka dependency, but if we have a simpler, cleaner way, let's go for it!